addon/components/o-s-s/link.hbs

```diff
@@ -0,0 +1,9 @@
+<a onclick="event.preventDefault()" href="" class="upf-link" ...attributes>
```

Comment:
What is the best option:
- use native js like in the code
- use `{{on 'click' this.preventDefault}}` and declare the action in ts file

I don't think that put functional logic in `hbs` is a good idea for the readability